src/index.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import './style.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-
+import  Header from './componentes/header';
+import Main from './componentes/main';
+import Wrapper from './componentes/wrapper';
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Wrapper />
```

Comment:
revert this change and create a folder for pages and import the page in app.js 
```suggestion
    <App />
```